startup/routes.js

```diff
@@ -1,8 +1,26 @@
 const express = require('express')
+const rateLimit = require('express-rate-limit')
+
+const ApiError = require('../helpers/apiError')
+const globalErrorHandler = require('../middleware/globalErrorHandler')
+
 const example = require('../routes/example')
 
 module.exports = function (app) {
   app.use(express.json())
 
-  app.use('/api/users', example)
+  const limit = rateLimit({
+    max: 100,
+    windowMs: 60 * 60 * 1000,
+    message: 'Too Many Requests',
+  })
+
+  app.use('/api', limit)
+  app.use('/api/v1/users', example)
+
+  app.use('*', (req, res, next) => {
+    next(new ApiError(404, 'Route is not supported.'), req, res, next)
```

Comment:
cant we just do?
res.status(404).send('Route is not supported.')

Reply:
Maybe, this way it will be handled by the globalErrorHandler and there it returns json and maybe later we will want to add something more.

Reply:
Okay, I think if we have time we will change it to Winston anyway. Only if we have time